indicator/src/main/java/com/fuzz/indicator/LayoutLogger.java

```diff
@@ -0,0 +1,92 @@
+package com.fuzz.indicator;
+
+import android.annotation.TargetApi;
+import android.os.Build;
+import android.support.annotation.NonNull;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import static android.os.Build.VERSION.SDK_INT;
+import static android.os.Build.VERSION_CODES.KITKAT;
+
+/**
+ * Accessible variant of
+ * {@code com.android.ide.common.rendering.api.LayoutLog}.
+ * <p>
+ *     The original class can not be linked against from
+ *     within Android Studio; this class uses reflection
+ *     to access and call methods on that class.
+ * </p>
+ *
+ * @author Philip Cohn-Cort (Fuzz)
+ */
+public abstract class LayoutLogger {
```

Comment:
Check out whether we can even test this code - it might not be possible to run unit or integration tests within the BridgeContext, at least not in the usual style.